apache/superset
# chore: Clean up the examples dashboards

### SUMMARY
This PR cleans up the examples dashboards by adjusting the dashboards layouts and removing:
- All filter boxes 
- 2 charts with incorrect metadata
- Charts that were previously removed

Fixes https://github.com/apache/superset/issues/26133

### TESTING INSTRUCTIONS
1 - Make sure the command to load the examples runs successfully
2 - Make sure there's no filter box
3 - Make sure all charts render correctly
4 - Make sure there are no deleted charts

### ADDITIONAL INFORMATION
